config/environments/test.rb

```diff
@@ -29,7 +29,7 @@
   config.cache_store = :null_store
 
   # Raise exceptions instead of rendering exception templates.
-  config.action_dispatch.show_exceptions = true
+  config.action_dispatch.show_exceptions = :all
```

Comment:
Fixes a warning that appeared following an upgrade to rails: `Setting action_dispatch.show_exceptions to true is deprecated. Set to :all instead.`

There is another warning that I believe is a devise issue and will be [fixed on their end](https://github.com/heartcombo/devise/issues/5644)